OpenRastaSwagger/Discovery/Heuristics/DiscoverHttpMethodVerbs.cs

```diff
@@ -23,6 +24,20 @@ public bool Discover(MethodInfo publicMethod, OperationMetadata methodMetdata)
             if (methodAttribute != null)
             {
                 methodMetdata.HttpVerb = methodAttribute.Method;
```

Comment:
It may be that the method selecting mechanism could do with a refactor in general, rather than the bandaid placed here, but at least now there are some tests to cover the scenario